leftwm-core/src/handlers/window_handler.rs

```diff
@@ -439,10 +439,10 @@ pub fn scratchpad_xyhw(xyhw: &Xyhw, scratch_pad: &ScratchPad) -> Xyhw {
 fn sane_dimension(config_value: Option<Size>, default_percent: f32, max_pixel: i32) -> i32 {
     match config_value {
         Some(size) => match size {
-            Size::Percentage(percentage) if (0.0..0.9).contains(&percentage) => {
+            Size::Percentage(percentage) if (0.0..=1.0).contains(&percentage) => {
                 size.into_absolute(100.0) as i32 * max_pixel / 100
             }
-            Size::Pixel(pixel) if (0..(max_pixel as f32 * 0.9) as i32).contains(&pixel) => pixel,
+            Size::Pixel(pixel) if (0..=max_pixel).contains(&pixel) => pixel,
             _ => (default_percent * max_pixel as f32) as i32,
         },
         _ => (default_percent * max_pixel as f32) as i32,
```

Comment:
And again... sorry to point out lines this PR didn't even touch.
Is there an easy way to prevent the duplicate code on line 446 and 448?

Reply:
I don't think so, unless it is refactored into if let... maybe.

Reply:
Fixed in 588faba